src/components/RefineSerach.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { Box, Text, Button } from '@chakra-ui/react'
+import { useSelector } from 'react-redux';
+import Router, { useRouter } from 'next/router';
+import { RootState } from '../../src/redux/store';
+import Link from 'next/link';
+
+
+
+const RefineSerach = () => {
+    const CategoryValues = useSelector((state: RootState) => state.categories.data);
```

Comment:
Always use camelCase for variables, Capitalized should be only for Components

```suggestion
    const categoryValues = useSelector((state: RootState) => state.categories.data);
```